tool/build.dart

```diff
@@ -176,15 +201,21 @@ _main(List<String> argv) async {
 
     await pushd("tmp/${rname}");
 
+    var branch = automated["branch"];
+
+    if (branch == null) {
+      branch = "HEAD";
```

Comment:
This allows you to configure a branch to build rather than just the default. HEAD is used here to refer to the root branch. You can't use `HEAD` with `git clone -b`, so we must work around that below.